### PR TITLE
Adds glorf jumpsuits as an assistant exclusive 20tc traitor item

### PR DESCRIPTION
**Pull request by anconfuzedrock**
:cl: rock
:add: Traitor assistants can now get an ancient jumpsuit for 20 tc. about as useful as a balloon.
/:cl:

### CODE REVIEW

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1208,6 +1208,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+/datum/uplink_item/role_restricted/ancient_jumpsuit
+	name = "Ancient Jumpsuit"
+	desc = "A uh, jumpsuit recovered from a station overcome by the grey tide. yeah."
```
Comment (MrPerson) [inline code/modules/uplink/uplink_item.dm]: That yeah is supposed to be capitalized.

Comment (anconfuzedrock) [inline code/modules/uplink/uplink_item.dm]: that good enough for ya?

**Comment by jughu** [comment]
>not just buying chamkit USELESS

**Comment by PKPenguin321** [comment]
This seems kind of misleading, these are about as useful as a syndicate balloon but aren't listed with it

**Comment by anconfuzedrock** [comment]
You think I should put it in the exclusives section AND the "for showing you are the boss" section?

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -1208,6 +1208,12 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 20
 	restricted_roles = list("Chaplain")
 	surplus = 5 //Very low chance to get it in a surplus crate even without being the chaplain
+/datum/uplink_item/role_restricted/ancient_jumpsuit
+	name = "Ancient Jumpsuit"
+	desc = "A jumpsuit recovered from a station overcome by the grey tide. Yeah."
```
Comment (KorPhaeron) [inline code/modules/uplink/uplink_item.dm]: Please give it an actual description so people know its a joke item and dont waste all their points and get mad

**Comment by Fox-McCloud** [comment]
I'm not sure if this even works.

**Comment by anconfuzedrock** [comment]
I can't think of a way to improve the description further without it soundin garbage and what do you mean it won't work 
if anyone can come up with a better description I'd love to hear it

**Comment by FantasticFwoosh** [comment]
Glorf jumpsuit should race change you into a glorf, inflicting clumsyness, undisguisable voice & a lack of exosuit & headslots just to be insulting when you dunk them. Not least to say you are also utmost valid to anybody.

#15760

Changeling spends TC to become a glorf (which is restricted from being identity stolen) then changes other players into the same glorf

**Comment by anconfuzedrock** [comment]
this is ready I think in case it wasn't clear

**Comment by DoctorPork** [comment]
It should turn you into Bryce Pax when you put it on.

**Comment by anconfuzedrock** [comment]
no shut up is there any conflict or something im missing here because its r e a d y

**Comment by anconfuzedrock** [comment]
great thanks now I don't have to blame myself for the description being shit

**Comment by KorPhaeron** [comment]
Honestly this is just unfunny bloat that brings nothing to the game

**Comment by anconfuzedrock** [comment]
I guess I just really wanted to give assistants something

**Pull request merged**